lib/process-services/task-list/components/start-task.component.ts

```diff
@@ -81,6 +85,7 @@ export class StartTaskComponent implements OnInit {
     }
 
     ngOnInit() {
+        this.field = new FormFieldModel(new FormModel(), {id: 'fakeId', value: 'fakeValue', placeholder: 'Assignee'});
```

Comment:
would be better remove the fakeId and use assigneeId 
same for value you can use assigneeId I guess

Reply:
I think this highlights a problem of trying to use this widget here . If this is needed like that wouldn't it be better try to create a common one that could be extended by the widget(which has the form) and the one you need?